src/libsystemd-network/dhcp-lease-internal.h

```diff
@@ -75,6 +75,8 @@ struct sd_dhcp_lease {
         uint16_t mtu; /* 0 if unset */
 
         char *domainname;
+        char **search_domains;
+        size_t search_domains_count;
```

Comment:
That said, why keep the counter anyway? the array itself should suffice, if we maintain it as the usual NULL-terminated strv?